### PR TITLE
Re-export bitraversable

### PR DESCRIPTION
**Pull request by tranma**
Is this ok? These are fairly common and have equivalents in Functor.

### CODE REVIEW

**Comment by markhibberd** [comment]
I am good 👍

**Comment by jystic** [comment]
What's the difference between `bifor` and `biforM` ?

**Comment by charleso** [comment]
@jystic @tranma Might be good to do something with this? Seems like we don't need the `M` versions any more?

**Comment by jystic** [comment]
yeah I'm happy for this, as long as we don't have duplicate functions

**Comment by tranma** [comment]
yeah we only need the Applicative versions, I'll just export Bitraversable, bisequenceA, bifor

**Comment by jystic** [comment]
worth exporting `bifor_` and `bitraverse_` from `Data.Bifoldable` as well?

**Comment by tranma** [comment]
yeah, let's

File: src/P.hs [hunk 1]
```diff
@@ -54,6 +54,11 @@ import           Data.Bitraversable as X (
                    , bisequenceA
                    , bifor
                    )
+import           Data.Bifoldable as X (
+                     Bifoldable (..)
+                   , bisequence_
+                   , bifor_
+                   )
```
Comment (jystic) [inline src/P.hs]: I think this will be missing [bitraverse_](http://hackage.haskell.org/package/bifunctors-5.4.1/docs/Data-Bifoldable.html#v:bitraverse_)

**Pull request merged**